daemon/snap.go

```diff
@@ -55,27 +55,27 @@ func snapDate(info *snap.Info) time.Time {
 }
 
 // localSnapInfo returns the information about the current snap for the given name plus the SnapState with the active flag and other snap revisions.
-func localSnapInfo(st *state.State, name string) (info *snap.Info, active bool, err error) {
+func localSnapInfo(st *state.State, name string) (info *snap.Info, snapstOut *snapstate.SnapState, err error) {
 	st.Lock()
 	defer st.Unlock()
 
 	var snapst snapstate.SnapState
```

Comment:
that is, the variable declared as part of the function declaration above. In case you didn't see the point the first time, like  I didn't.